### PR TITLE
DMU 9 Cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

**Comment by Agetian** [comment]
Looks good to me when it comes to AI hints and basic structure, but I could use some help reviewing the mechanics :)

File: forge-gui/res/cardsfolder/upcoming/choking_miasma.txt [hunk 1]
```diff
@@ -3,6 +3,6 @@ ManaCost:1 B B
 Types:Sorcery
 K:Kicker:G
 A:SP$ PumpAll | ValidCards$ Creature | NumAtt$ -2 | NumDef$ -2 | SubAbility$ PutCounter | SpellDescription$ If this spell was kicked, put a +1/+1 counter on a creature you control. All creatures get -2/-2 until end of turn.
-SVar:PutCounter:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | CounterType$ P1P1 | Condition$ Kicked | CounterNum$ 1
+SVar:PutCounter:DB$ PutCounter | Choices$ Creature.YouCtrl | CounterType$ P1P1 | Condition$ Kicked | CounterNum$ 1
```
[the remark this answers is not in the document]
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/choking_miasma.txt:6]: Also you may want to add ChoiceTitle$ depending on how the prompt looks

File: forge-gui/res/cardsfolder/upcoming/flowstone_infusion.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Name:Flowstone Infusion
+ManaCost:R
+Types:Instant
+A:SP$ Pump | ValidTgts$ Creature | NumAtt$ +2 | NumDef$ -2 | SpellDescription$ Target creature gets +2/-2 until end of turn.
+AI:RemoveDeck:Random
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/flowstone_infusion.txt:5]: Not sure this is needed here – similar things like Bedeck and Flowstone Armor don't have this AI indication

File: forge-gui/res/cardsfolder/upcoming/ghitu_amplifier.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Ghitu Amplifier
+ManaCost:1 R
+Types:Creature Human Wizard
+PT:1/2
+K:Kicker:2 U
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self+kicked  | Execute$ TrigReturn | TriggerDescription$ When CARDNAME enters the battlefield, return target creature an opponent controls to its owner's hand.
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Creature.OppCtrl | TargetMax$ 1 | Origin$ Battlefield | Destination$ Hand
+T:Mode$ SpellCast | ValidCard$ Instant,Sorcery | ValidActivatingPlayer$ You | TriggerZones$ Battlefield | Execute$ DBPump | TriggerDescription$ Whenever you cast an instant or sorcery spell, CARDNAME gets +2/+0 until end of turn.
+SVar:DBPump:DB$ Pump | Defined$ Self | NumAtt$ 2
+DeckHints:Type$Instant|Sorcery
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ghitu_amplifier.txt:10]: ` & Color$Blue`

File: forge-gui/res/cardsfolder/upcoming/choking_miasma.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Choking Miasma
+ManaCost:1 B B
+Types:Sorcery
+K:Kicker:G
+A:SP$ PumpAll | ValidCards$ Creature | NumAtt$ -2 | NumDef$ -2 | SubAbility$ PutCounter | SpellDescription$ If this spell was kicked, put a +1/+1 counter on a creature you control. All creatures get -2/-2 until end of turn.
+SVar:PutCounter:DB$ PutCounter | Choices$ Creature.YouCtrl | CounterType$ P1P1 | Condition$ Kicked | CounterNum$ 1
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/choking_miasma.txt:6]: `DeckHints:Color$Green`

File: forge-gui/res/cardsfolder/upcoming/ghitu_amplifier.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Ghitu Amplifier
+ManaCost:1 R
+Types:Creature Human Wizard
+PT:1/2
+K:Kicker:2 U
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self+kicked  | Execute$ TrigReturn | TriggerDescription$ When CARDNAME enters the battlefield, return target creature an opponent controls to its owner's hand.
+SVar:TrigReturn:DB$ ChangeZone | ValidTgts$ Creature.OppCtrl | TargetMax$ 1 | Origin$ Battlefield | Destination$ Hand
```
Comment (Northmoc) [inline forge-gui/res/cardsfolder/upcoming/ghitu_amplifier.txt:7]: Add `TgtPrompt$ Select target creature an opponent controls`

Remove TargetMax

File: forge-gui/res/cardsfolder/upcoming/ghitu_amplifier.txt (hunk 3 above)
Comment (Northmoc) [reply forge-gui/res/cardsfolder/upcoming/ghitu_amplifier.txt:10]: `SVar:BuffedBy:Instant,Sorcery`

**Review by tool4ever — APPROVED** [review]

**Pull request merged**